src/views/ModelPlan/TaskList/Basics/Overview/index.tsx

```diff
@@ -0,0 +1,333 @@
+import React, { useRef, useState } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { Link, useHistory, useParams } from 'react-router-dom';
+import { useMutation, useQuery } from '@apollo/client';
+import {
+  Breadcrumb,
+  BreadcrumbBar,
+  BreadcrumbLink,
+  Button,
+  Fieldset,
+  IconAdd,
+  IconArrowBack,
+  Label,
+  Radio
+} from '@trussworks/react-uswds';
+import { Field, Form, Formik, FormikProps } from 'formik';
+
+import MainContent from 'components/MainContent';
+import PageHeading from 'components/PageHeading';
+import PageNumber from 'components/PageNumber';
+import AutoSave from 'components/shared/AutoSave';
+import { ErrorAlert, ErrorAlertMessage } from 'components/shared/ErrorAlert';
+import FieldErrorMsg from 'components/shared/FieldErrorMsg';
+import FieldGroup from 'components/shared/FieldGroup';
+import TextAreaField from 'components/shared/TextAreaField';
+import GetModelPlanQuery from 'queries/GetModelPlanQuery';
+import {
+  GetModelPlan,
+  GetModelPlanVariables
+} from 'queries/types/GetModelPlan';
+import { UpdatePlanBasics as UpdatePlanBasicsType } from 'queries/types/UpdatePlanBasics';
+import UpdatePlanBasics from 'queries/UpdatePlanBasics';
+import flattenErrors from 'utils/flattenErrors';
+import planBasicsSchema from 'validations/planBasics';
+
+interface PlanBasicsOverviewTypes {
+  modelType: string;
+  problem: string;
+  goal: string;
+  testInterventions: string;
+  note: string;
+}
+
+const Overview = () => {
+  const { t } = useTranslation('basics');
+  const { t: h } = useTranslation('draftModelPlan');
+  const { modelId } = useParams<{ modelId: string }>();
+
+  const formikRef = useRef<FormikProps<PlanBasicsOverviewTypes>>(null);
+  const history = useHistory();
+  const [hasAdditionalNote, setHasAdditionalNote] = useState(false);
+
+  const { data } = useQuery<GetModelPlan, GetModelPlanVariables>(
+    GetModelPlanQuery,
+    {
+      variables: {
+        id: modelId
+      }
+    }
+  );
+
+  const { modelName } = data?.modelPlan || {};
+  const { id, modelType, problem, goal, testInventions, note } =
+    data?.modelPlan?.basics || {};
+
+  const [update] = useMutation<UpdatePlanBasicsType>(UpdatePlanBasics);
+
+  const initialValues = {
+    modelType: modelType ?? '',
+    problem: problem ?? '',
+    goal: goal ?? '',
+    testInterventions: testInventions ?? '',
+    note: note ?? ''
+  };
+
+  const handleFormSubmit = (
+    formikValues: PlanBasicsOverviewTypes,
+    isAutoSave: boolean = false,
```

Comment:
I think a rework of how we handle redirects on all types of form submissions is needed.  Instead of passing multple props to the submission, I suggest a single prop like `redirect`.  It can be optional and it would look something like this - 

`redirect?: 'next' | 'back' | 'task-list'`

Where the absence of the prop would signal an autosave, and other submissions pass in one of those string props to tell where to redirect after successful mutation.